assets/pages/contributions-landing/components/contributionsBundle.jsx

```diff
@@ -66,31 +66,39 @@ type ContribAttrs = {
 
 // ----- Copy ----- //
 
-const subHeadingMonthlyText = {
-  GB: 'from £5 a month',
-  US: 'from $5 a month',
-  AU: 'from $5 a month',
+const subHeadingMonthlyText: {[IsoCurrency]: string} = {
+  GBP: 'from £5 a month',
+  USD: 'from $5 a month',
+  AUD: 'from $5 a month',
+  EUR: 'from €5 a month',
 };
 
-function getSubHeadingMonthly(abTests: Participations, isoCountry: IsoCountry) {
-  return subHeadingMonthlyText[isoCountry];
+function getSubHeadingMonthly(abTests: Participations, isoCurrency: IsoCurrency) {
+  return subHeadingMonthlyText[isoCurrency];
 }
```

Comment:
Does this need to be a function? Can it just be an object lookup at the call point? By the looks it was added for an AB test, but the `abTests` argument isn't used any more.

Reply:
Maybe, but this PR is to big to do refactor this here. I will do a refactor PR soon.